deploy/esignet/install.sh

```
@@ -102,8 +102,7 @@ function installing_esignet() {
             --set persistence.mountDir=\"$volume_mount_path\" \
             --set persistence.pvc_claim_name=\"$PVC_CLAIM_NAME\"  \
             --set extraEnvVarsCM={'esignet-global','config-server-share','artifactory-share'} \
-            --set extraEnvVarsAdditional[0].name="MOSIP_KERNEL_KEYMANAGER_HSM_KEYSTORE-TYPE" \
-            --set extraEnvVarsAdditional[0].value="PKCS12" \
+            --set extraEnvVarsAdditional.MOSIP_KERNEL_KEYMANAGER_HSM_KEYSTORE-TYPE="PKCS12" \
```

Shellcheck (0.11.0)

[warning] Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?

(SC2140)